staging/src/k8s.io/client-go/util/flowcontrol/backoff.go

```diff
@@ -17,24 +17,38 @@ limitations under the License.
 package flowcontrol
 
 import (
+	"math/rand"
 	"sync"
 	"time"
 
 	"k8s.io/apimachinery/pkg/util/clock"
 	"k8s.io/utils/integer"
 )
 
+// TODO(vllry) consider allowing this to be customized in NewBackOff.
+const DefaultJitterRatio float64 = 0.20 // 20%
```

Comment:
Since folks can't change it, it's not really a default, it's just the `JitterRatio`. Does it need to be public?

Reply:
It was exported for the purpose of tests - I'll reevaluate if it's necessary in that context.